scripts/publish-dryrun.sh

```diff
@@ -0,0 +1,82 @@
+ #!/usr/bin/env bash
```

Comment:
everything from 1-81 is duplicated here and in smoke-tests.sh this could be deduped into another script